src/app/pages/sign-transaction/sign-transaction.tsx

```diff
@@ -92,9 +94,13 @@ function SignTransactionBase(): JSX.Element | null {
         validateOnChange={false}
         validateOnBlur={false}
         validateOnMount={false}
-        validationSchema={yup.object({
-          fee: feeSchema(),
-        })}
+        validationSchema={
+          !isSponsored
+            ? yup.object({
+                fee: feeSchema(),
+              })
+            : null
+        }
```

Comment:
Would recommend moving this out of the JSX inline definition now it's conditional